app/models/katello/concerns/http_proxy_extensions.rb

```diff
@@ -11,9 +11,10 @@ def update_default_proxy_setting
         changes = self.previous_changes
         if changes.key?(:name)
           previous_name = changes[:name].first
-          setting = Setting.
+          content_settings = Setting::Content.arel_table
+          setting = Setting::Content.unscoped.
             where(name: "content_default_http_proxy").
-            where(value: previous_name.to_yaml).
+            where(content_settings[:value].matches("%#{previous_name}%")).
```

Comment:
this seems kinda wrong as 'foo' would match 'foobar'.  What about something like:
```
setting = Setting::Content.unscoped.find_by(name: "content_default_http_proxy")
if setting.value == previous_name:
  UPDATE_NAME
end
```